otbenchmark/ReliabilityProblem110.py

```diff
@@ -60,9 +60,9 @@ def __init__(self, threshold=0.0, mu1=0.0, sigma1=1.0, mu2=0.0, sigma2=1.0):
         program = "\n".join(equations)
         print(program)
         limitStateFunction = ot.SymbolicFunction(["x0", "x1"], ["gsys"], program)
-        X1 = ot.Normal(mu1, sigma1)
+        X1 = ot.Normal(mu[0], sigma[0])
```

Comment:
Please check the length of the mu list. If it is not equal to 2, raise an error.

Please apply this change to the other classes as well.

Reply:
I did not understand, However, For the PR33, we have the length of mu is 3 !!

Reply:
This comment is for RP110, which has two gaussians. More generally, if the problem has n gaussian, then an exception must be raised when the number of items in the list is different from n. Please look at https://github.com/mbaudin47/otbenchmark/blob/e92e5a8a3e66264516c33ee4e24edaddafa017d7/otbenchmark/DrawEvent.py#L77 as an example of a situation where we raise such an error.

Reply:
This is correct.